backend/public-graph/graph/resolver.go

```diff
@@ -157,6 +159,14 @@ type NetworkResource struct {
 	RequestResponsePairs RequestResponsePairs `json:"requestResponsePairs"`
 }
 
+type LoginCredentials struct {
+	Email    string `json:"email,omitempty"`
+	Password string `json:"password"`
+}
+
+var JwtAccessSecret = os.Getenv("JWT_ACCESS_SECRET")
+var ADMIN_PASSWORD = os.Getenv("ADMIN_PASSWORD")
```

Comment:
could we move this and the `Login` method below to a separate `private-graph/graph/auth.go` (or the middleware.go) file that's part of the private-graph `graph` package? would make it cleaner for these to be next to the login handler in a separate auth related file

Reply:
Done